### ISSUE TITLE
88xx bug

### ISSUE THREAD

**Issue by pilotnbr1** [RespawnDespair/wifibroadcast-image-builder#106]
In an effort to document and stay organized it pains me to open this 😔

**Comment by pilotnbr1** [comment]
Just started looking into the 88xx problem. Anyone aware of some of the problems reported by 5.3.4? Sorry if this is already mentioned. I am just getting up to speed..
https://github.com/t6x/reaver-wps-fork-t6x/issues/265#issuecomment-441272085

**Comment by Yes21** [comment]
The issue is : with 5.3.4 on the airPi, no video transmission to the ground and error message "injection failed".

**Comment by RespawnDespair** [comment]
Its definitely a better decision to move back to 5.2.20, there appear to be more people with issues (in regards to monitor mode etc.):
https://github.com/t6x/reaver-wps-fork-t6x/issues/265
I will try to go back while mainting all the current improvements.

**Comment by pilotnbr1** [comment]
To go back to 5.2.20 is it as simple as changing the git request? Or is it more involved than it appears?

**Comment by RespawnDespair** [comment]
A little more involved, since the rest of the modifications still needs to work

**Comment by pilotnbr1** [comment]
@JelleTigchelaar just want to point out that the makefile for 5.2.20 now (as of a week ago) has option specific to pi3b+

CONFIG_PLATFORM_ARM_RPI = n
CONFIG_PLATFORM_ARM64_RPI = n

**Comment by pilotnbr1** [comment]
So to get to the bottom of our missing modules I have attempted some fairly aggressive rollbacks on the code.

I have gone so far as to completely comment out all purges, auto-remove, completely deleted stage 4 04-run.sh. 

I am about to comment out 
DEBIAN_FRONTEND=noninteractive sudo apt-get -yq install raspberrypi-kernel-headers

Comparing @Yes21 11-05 image there are quite a few more drivers present within /rootfs/lib/modules/4.14.71-v7+/kernel/drivers than any new image I create. lsmod on a live image also confirms missing modules in the newer images

Ideas? not sure what changed to screw this up

**Comment by pilotnbr1** [comment]
Comparing the stage 2 kernel config file in @JelleTigchelaar repo and my older cloned repo there is a difference at line 1170.

my repo-
# CONFIG_RTL8XXXU is not set
CONFIG_RTL8812AU=y

this repo-
# CONFIG_RTL8XXXU is not set
# CONFIG_RTL8812AU=y
CONFIG_RTL8821AU=m

Its commented out. dont know if the patch comes later or some other process changes it down the road but it is a difference in the 2 repo files.

**Comment by Yes21** [comment]
I can confirm that the latest commit (https://github.com/RespawnDespair/wifibroadcast-image-builder/commit/ab0206dba7e320df7d9407c91319366f90717693) does this #CONFIG_RTL8812AU=y.

But @RespawnDespair did explain me that's not an error :

> Yves Chaufour
Jelle I've found 3 errors were you wrote 8821 instead of 8812. But now the driver is not compiling anymore !!
Jelle Tigchelaar
Those are not errors. The entire set if modules us 8821au to the kernel config. Within the makefile for the 88xx you determine the actual support. So i dont think that is a problem. The gnd loads the driver for you 8812 doesnt it?

**Comment by pilotnbr1** [comment]
@Yes21 thanks for the reply. I am sure @JelleTigchelaar is right then...  I am running out of ideas and just going earlier in the stages to figure it out.

I’ve already been building an image with older config files (that appear correct) to see what that does. Sounds like it will not fix the problem

**Comment by Yes21** [comment]
But I don't understand why @RespawnDespair did change to 8821 for the v5.3.4 branch of the 8812 driver ...

**Comment by kimocoder** [comment]
Pushing both major and minor updates to the v5.2.20 branch within a week.
Rolling out and may be seen here https://github.com/aircrack-ng/rtl8812au/pull/246

when I finish up, I'll port them into the v5.3.4 (Edimax) base.

contains frame injection fix and lots of other important fixes. so stay tuned

**Comment by Yes21** [comment]
> Pushing both major and minor updates to the v5.2.20 branch within a week.
> Rolling out and may be seen here [aircrack-ng/rtl8812au#246](https://github.com/aircrack-ng/rtl8812au/pull/246)
> 
> when I finish up, I'll port them into the v5.3.4 (Edimax) base.
> 
> contains frame injection fix and lots of other important fixes. so stay tuned

Many thanks.
I've now reverted our script to 5.2.20.
We will test again 5.3.4 later ...

**Comment by kimocoder** [comment]
Yup. v5.3.4 is good, but it got some issues still.. work in progres more or less

**Comment by Yes21** [comment]
Here is the real issue we had with 5.3.4 :

> The issue is : with 5.3.4 on the airPi, no video transmission to the ground and error message "injection failed".

**Comment by pilotnbr1** [comment]
I think we can close this...
